executor/write.go

```diff
@@ -682,6 +683,7 @@ func (e *InsertExec) Close() error {
 	return nil
 }
 
+// getColumns ...
```

Comment:
getColumns gets the explicitly specified columns of an insert statement.  There are three cases: